app/ide-desktop/lib/content/src/index.ts

```diff
@@ -180,6 +180,10 @@ class Config {
         'Controls whether the new component browser should be enabled.'
     )
     emitUserTimingMeasurements: config.Param<boolean> = new config.Param(false, 'TODO')
+    enableSkipAndFreeze: config.Param<boolean> = new config.Param(
+        false,
+        'Enable experimental support for SKIP and FREEZE macros.'
```

Comment:
I'm not sure if we should say users about _macros_. Maybe "support for skipping and freezing nodes"?